src/Microsoft.AspNetCore.Http.Extensions/UriHelper.cs

```diff
@@ -195,10 +195,10 @@ public static string GetEncodedPathAndQuery(this HttpRequest request)
         /// <returns></returns>
         public static string GetDisplayUrl(this HttpRequest request)
         {
-            var host = request.Host.Value;
-            var pathBase = request.PathBase.Value;
-            var path = request.Path.Value;
-            var queryString = request.QueryString.Value;
+            var host = request.Host.Value ?? "";
+            var pathBase = request.PathBase.Value ?? "";
+            var path = request.Path.Value ?? "";
+            var queryString = request.QueryString.Value ?? "";
 
             // PERF: Calculate string length to allocate correct buffer size for StringBuilder.
             var length = request.Scheme.Length + SchemeDelimiter.Length + host.Length
```

Comment:
Might as well make the same change for scheme, it's the only one not following the pattern.